src/esbmc/options.cpp

```diff
@@ -482,7 +482,8 @@ const struct group_opt_templ all_cmd_options[] = {
     {"goto-coverage",
      NULL,
      "this activates --make-assert-false and --multi-property and "
```

Comment:
Please replace the last "and" in this line with a comma - for readability. Lots of "and" in this sentence. ;)

Reply:
Will change